include/pybind11/pybind11.h

```diff
@@ -1731,6 +1732,7 @@ template <typename Type> class enum_ : public class_<Type> {
         m_base.init(is_arithmetic, is_convertible);
 
         def(init([](Scalar i) { return static_cast<Type>(i); }), arg("value"));
+        def_property("value", [](Type value) { return (Scalar) value; }, nullptr);
```

Comment:
Just use `def_property_readonly`. No need to introduce `def_property`.

Reply:
Done.